exercises/practice/bob/.approaches/answer-array/snippet.txt

```diff
@@ -0,0 +1,8 @@
+var input = statement.TrimEnd();
+if (input == "") return "Fine. Be that way!"; 
+
+var isShout = (input.Any(c => char.IsLetter(c)) && input.ToUpper() == input) ? 2: 0;    
+var isQuestion = (input.EndsWith('?')) ? 1: 0;    
```

Comment:
We could get rid of the unneeded parentheses, like we did in the `content.md` and use a method group for `char.IsLetter`:

```suggestion
var isShout = input.Any(char.IsLetter) && input.ToUpper() == input ? 2: 0;    
var isQuestion = input.EndsWith('?') ? 1: 0;    
```